runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkBatchTransformTranslators.java

```diff
@@ -640,7 +640,8 @@ public void translateNode(
               windowingStrategy,
               sideInputStrategies,
               context.getPipelineOptions(),
-              outputMap);
+              outputMap,
```

Comment:
This file has three references to `OldFoFn` in comments. These should be changed to just `DoFn`.